build.cmd

```diff
@@ -11,12 +11,12 @@ if NOT "%VSINSTALLDIR%"=="" goto InDevPrompt
 set x86ProgramFiles=%ProgramFiles(x86)%
 if "%x86ProgramFiles%"=="" set x86ProgramFiles=%ProgramFiles%
 set VSWherePath=%x86ProgramFiles%\Microsoft Visual Studio\Installer\vswhere.exe
-if NOT exist "%VSWherePath%" echo ERROR: Could not find vswhere.exe (%VSWherePath%). Ensure that Visual Studio 2017 version 15.5 or newer is installed. & exit /b -1 
+if NOT exist "%VSWherePath%" echo ERROR: Could not find vswhere.exe (%VSWherePath%). Ensure that Visual Studio 2022 is installed. & exit /b -1 
 
-for /f "usebackq tokens=1 delims=" %%a in (`"%VSWherePath%" -version [16.0, -prerelease -requires Microsoft.VisualStudio.Workload.NativeDesktop;Microsoft.VisualStudio.Workload.VisualStudioExtension;Microsoft.VisualStudio.Workload.ManagedDesktop -property installationPath`) do call :ProcessIDE "%%a"
+for /f "usebackq tokens=1 delims=" %%a in (`"%VSWherePath%" -version [17.0, -prerelease -requires Microsoft.VisualStudio.Workload.NativeDesktop;Microsoft.VisualStudio.Workload.VisualStudioExtension;Microsoft.VisualStudio.Workload.ManagedDesktop -property installationPath`) do call :ProcessIDE "%%a"
 if NOT "%VSINSTALLDIR%"=="" goto InDevPrompt
 
-echo ERROR: Unable to find a Visual Studio install.
+echo ERROR: Unable to find a Visual Studio 2022 or newer install.
```

Comment:
nit: you deleted "or newer" above.  I assume you don't want it here either?

Reply:
Actually, I didn't mean to delete the 'or newer' above. I meant to just delete the fact that 15.5 was okay. Let me put the "or newer" back.